server/service/FlightService.js

```diff
@@ -456,7 +456,7 @@ const flightService = {
     const fixesStats = attachFlightStats(flight, fixes);
     await storeFixesToDB(flight, fixes, fixesStats);
   },
-
+  // TODO: It's confusing that the method returns the takeoff…
```

Comment:
If you need the flyingSite now as whole outside of this function we can split this function into two...

Reply:
War aber schon vorher verwirrend:P